samples/common/mcumgr_bt_ota_dfu/Kconfig

```diff
@@ -138,6 +138,7 @@ config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_VALIDATION
 
 config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_MCUBOOT_DIRECT_XIP_SUPPORT
```

Comment:
Can we remove the `imply MCUMGR_GRP_IMG_DIRECT_UPLOAD` imply according to the discussion with @de-nordic?